.github/workflows/build.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       - name: Setup Node.js
         uses: actions/setup-node@v2
```

Comment:
Should this be upgraded to v4?

Reply:
I tried v4 but got a permission error (actions/checkout@v4 and actions/setup-node@v4 are not allowed to be used in aws/awsui-documentation) about:
https://github.com/aws/awsui-documentation/actions/runs/11589513297

After some research, without successs, about how to change that, I stopped and moved on with this PR.

Reply:
Updated repo settings. You should be able to use v4 for both checkout and setup-node now.